src/app/parcels/parcelsTable/filters.ts

```diff
@@ -19,17 +19,13 @@ import { buildServerSideTextFilter } from "@/components/Tables/TextFilter";
 import dayjs from "dayjs";
 import { parcelTableHeaderKeysAndLabels } from "./headers";
 import { DbParcelRow } from "@/databaseUtils";
-import {
-    fullNameMethodHelper,
-    phoneNumberMethodHelper,
-    postcodeMethodHelper,
-} from "@/common/databaseFilters";
+import { fullNameSearch, phoneSearch, postcodeSearch } from "@/common/databaseFilters";
 
-const fullNameSearch: ParcelsFilterMethod<string> =
-    fullNameMethodHelper<DbParcelRow>("client_full_name");
+const parcelsFullNameSearch: ParcelsFilterMethod<string> =
```

Comment:
let's be consistent between parcels vs parcel - i don't really have a preference but just stick with one :)